src/web/lib/components/App/index.js

```diff
@@ -12,27 +12,30 @@ import ExtensionInstallButton from '../ExtensionInstallButton';
 import SharedThemeDialog from '../SharedThemeDialog';
 import AppLoadingIndicator from '../AppLoadingIndicator';
 import ThemeUrl from '../ThemeUrl';
+import ThemeSaveButton from '../ThemeSaveButton';
+import SavedThemeSelector from '../SavedThemeSelector';
 
 import './index.scss';
 
-const mapStateToProps = state => ({
-  theme: selectors.theme(state),
-  themeCanUndo: selectors.themeCanUndo(state),
-  themeCanRedo: selectors.themeCanRedo(state),
-  hasExtension: selectors.hasExtension(state),
-  loaderDelayExpired: selectors.loaderDelayExpired(state),
-  selectedColor: selectors.selectedColor(state),
-  shouldOfferPendingTheme: selectors.shouldOfferPendingTheme(state),
-  pendingTheme: selectors.pendingTheme(state)
-});
+const mapStateToProps = state => {
+  const mappedSelectors = {};
```

Comment:
This might be a too-clever idea, but looked like the pattern here was to just keep adding straight mappings of all selectors to the state.